waffle-cli/src/cli.ts

```diff
@@ -1,8 +1,12 @@
 import {compileProject} from '@ethereum-waffle/compiler';
+import {flattenProject} from '@ethereum-waffle/compiler/src/flattener';
```

Comment:
Import should look like this:
```typescript
import {compileProject, flattenProject} from '@ethereum-waffle/compiler';
```

Add an export to `waffle-compiler/src/index.ts` to achieve this.